website/docs/installation/windows.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 8
+---
+# Windows
```

Comment:
Maybe it would be better to rename this file/title to "WinGet" instead for being more specific, what do you think?

Reply:
Possibly! Or windows.md, with a section for WinGet, then any other sections in the future? Up to you of course

Reply:
Thinking about it more, you're right, winget.md would be better!